travis-setup.md

```diff
@@ -0,0 +1,88 @@
+    - The resulting line should look like this:
+      - `firebase_bucket=gs://dracco1993-tba-dev-17792.appspot.com`
+
+### Create/Download Firebase keys
+ 1) On the top left of Firebase, click the gear icon, then go to Project Settings
+ 2) From here, go to the Service Accounts tab
+ 3) Click "Generate new private key", then confirm by clicking "Generate key"
+    - Generating a key in this manner will download a new `.json` file, containing all the information required to access this Firebase project via an API
+    - As the warning in Firebase says, "Keep it confidential and **never store it in a public repository**"
+
+### Encrypt Firebase keys with Travis CLI
+ 1) Setup the Travis CLI
+    - https://github.com/travis-ci/travis.rb#installation
+ 2) Inside this repository's directory:
+    ```bash
+    $ travis login
+    ```
+ 3) Encrypt Firebase keys
+    - At this point, we need to encrypt the `.json` file that we downloaded from Firebase previously.  Travis has a very nice way to do this, that allows us to upload the encrypted file inside our repository, but for only Travis to be able to decrypt it when it's actually running a build.
+      ```bash
+      $ travis encrypt-file PATH_TO_DOWNLOADED_FILE.json gcp-service-key.json.enc
+      ```
+    - This command encrypts the `.json` file from Firebase, and sets two environment variables in your Travis project
+    - Note: the input path will be to wherever you put the Firebase key file, but to keep things easier, the output path will be the encoded `.json.enc` file already in the repository
+    - The full documentation for this command is available here:
+      - https://github.com/travis-ci/travis.rb#encrypt-file
+
+### Update `.travis.yml` with new config
+ 1) There is one final place that we need to update in `.travis.yml` and that is the environment variables that Travis uses to decrypt the `.json.enc` we just created
+ 2) To get the names of the variables, run the following command:
+    ```
+    $ travis env list
+    # environment variables for dracco1993/the-blue-alliance-react
+    encrypted_cb3726d3b29b_key=[secure]
+    encrypted_cb3726d3b29b_iv=[secure]
+    ```
+ 3) Take the two environment variables listed above and replace the corresponding sections of the first command in the `before_install` step of `.travis.yml`
+    - `openssl aes-256-cbc -K ${ENCRYPTED_KEY_NAME} -iv ${ENCRYPTED_IV_NAME} -in gcp-service-key.json.enc -out gcp-service-key.json -d`
+    - In the example above, the resulting line would be:
+    - `openssl aes-256-cbc -K $encrypted_cb3726d3b29b_key -iv $encrypted_cb3726d3b29b_iv -in gcp-service-key.json.enc -out gcp-service-key.json -d`
+
+### Commit changes
+ 1) After following the previous steps, you should have two files that you've updated and need to commit to the repository:
+    - `.travis.yml`
+    - `gcp-service-key.json.enc`
+
+    NOTE: **make sure you DO NOT commit the unencrypted `.json` file**
```

Comment:
Let's add whatever the name of what this is to the .gitignore 😄

Reply:
The file name is auto-generated by Firebase, so we don't know what it will be.  I went ahead and added some ignores, including matchers that would exclude the filenames suggested by the tutorial, in 38c04fd00ea3a3b9a51955ea1885f36d0f91953f